### PR TITLE
Create user in hubspot before sign in

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?188760

@proteusvacuum not sure this is the best way to do this. since the login comes from a signal, but the register user did not, there'd need to be a larger refactor to use celery chaining. Or I could just be missing something obvious

cc: @esoergel

### CODE REVIEW

File: corehq/apps/analytics/signals.py [hunk 1]
```diff
@@ -132,4 +132,4 @@ def track_user_login(sender, request, user, **kwargs):
             return
 
         meta = get_meta(request)
-        track_user_sign_in_on_hubspot.delay(couch_user, request.COOKIES, meta)
+        track_user_sign_in_on_hubspot(couch_user, request.COOKIES, meta, request.path)
```
Comment (proteusvacuum) [inline corehq/apps/analytics/signals.py]: doesn't this make this synchronous? does this not work if it remains a task?

**Comment by proteusvacuum** [comment]
:+1: This is pretty much what I was thinking. Thanks! One comment, otherwise g2m. 
@calellowitz FYI.

File: corehq/apps/analytics/signals.py (hunk 1 above)
Comment (emord) [inline corehq/apps/analytics/signals.py]: Oops. I had that around for testing, but forgot to take it out

Comment (proteusvacuum) [inline corehq/apps/analytics/signals.py]: :+1: if you add `CELERY_ALWAYS_EAGER= True` to your localsettings I don't think you need to do that.

Comment (emord) [inline corehq/apps/analytics/signals.py]: For some reason that only works sometimes for me. I haven't tracked down
why though.

**Pull request merged**